### ISSUE TITLE
Some label Meanings，I want to see if my theory is correct

### ISSUE THREAD

**Issue by TangYH321** [alipay/Pyraformer#21]
Excuse me, I would like to ask you the following questions:

① What does the data set with the _lable_, _v_, and _data_ connectors under the data/elect file mean respectively?

② What does the single_step_main.py file contain "sigma" and "mu" labels?

If convenient, could you please leave a contact information for me to learn from you?

Thank you very much!

Looking forward to your reply.

**Comment by Zhazhan** [comment]
Hello, thank you for your interest in our work. Your questions are explained below:

① 'Data' represents the sequences of fixed length splitted over the original dataset. For the electricity dataset, the window length is 192. 'Label' represents the prediction target of each element in the sequence. 'V' represents the mean and standard deviation of each sequence. It is used to normalize the historical sequence.

② Follow DeepAR, we let the network predict the mean (mu) and standard deviation (sigma) of the next time step. Then, a Gaussian distribution is constructed using 'mu' and 'sigma', and the likelihood of the ground truth under this distribution is taken as the optimization target.

My personal email address is shanluzuode@sjtu.edu.cn. Feel free to email to me.

**Comment by mw66** [comment]
Actually, the the optimization target is the following two combined, and with the magic number 100 (why)?

https://github.com/ant-research/Pyraformer/blob/master/single_step_main.py#L113-L114
```
loss = likelihood_loss + 100 * mse_loss
loss.backward()
```

I'm just wondering if the magic number is chosen any number between [0 , 100], will the results be different?

Or, more generally:

```
  loss = a * likelihood_loss + b * mse_loss
```

How would you choose `a` and `b`? Any considerations?

**Comment by mw66** [comment]
@Zhazhan do you have time to answer my above questions? Thanks
